ibl_to_nwb/updated_conversion/datainterfaces/iblsortinginterface.py

```diff
@@ -1,13 +1,25 @@
-"""The interface for loadding spike sorted data via ONE access."""
+"""The interface for loading spike sorted data via ONE access."""
+from pathlib import Path
+
 from neuroconv.datainterfaces.ecephys.basesortingextractorinterface import (
     BaseSortingExtractorInterface,
 )
+from neuroconv.utils import load_dict_from_file
 
 from .iblsortingextractor import IblSortingExtractor
 
 
 class IblSortingInterface(BaseSortingExtractorInterface):
     Extractor = IblSortingExtractor
 
-    # def get_metadata(self):
-    #    pass  # TODO: add descriptions for all those custom properties
+    def get_metadata(self) -> dict:
+        metadata = super().get_metadata()
+
+        ecephys_metadata = load_dict_from_file(file_path=Path(__file__).parent.parent / "metadata" / "ecephys.yml")
+        metadata.update(ecephys_metadata)
+
+        if "allen_location" in self.sorting_extractor.get_property_keys():
+            for column_name in ["allen_location", "beryl_location", "cosmos_location"]:
+                metadata["Ecephys"]["UnitProperties"].extend(ecephys_metadata["Ecephys"][column_name])
```

Comment:
Will this work? I think it would be `ecephys_metadata["Ecephys"]["Electrodes"]` and I think it's a list.

Side note: should it be a list? Maybe this is another place where a dictionary would be more appropriate.

Reply:
Yep - fixed

It must be a list until we update NeuroConv to use dictionaries. Pinning conversion to NeuroConv release v0.2.5 right now